src/core/gltf2importer/gltf2exporter.cpp

```diff
@@ -235,23 +244,238 @@ class GLTF2DracoCompressor
         {
             auto it = primitive_json.find(GLTF2Import::KEY_INDICES);
             if (it != primitive_json.end() && it->isDouble()) {
-                compressed_primitive.accessors_to_clean.insert(it->toInt());
+                compressed_primitive.accessorsToClean.insert(it->toInt());
             }
 
             it = primitive_json.find(GLTF2Import::KEY_ATTRIBUTES);
             if (it != primitive_json.end() && it->isObject()) {
                 const auto attributes = it->toObject();
                 for (const auto &attr : attributes) {
                     if (attr.isDouble())
-                        compressed_primitive.accessors_to_clean.insert(attr.toInt());
+                        compressed_primitive.accessorsToClean.insert(attr.toInt());
                 }
             }
         }
-        compressed_primitive.primitive_json = primitive_json;
+        compressed_primitive.primitiveJson = primitive_json;
 
         return compressed_primitive;
     }
 
+    void removeBufferViews(const std::set<int> &indices)
+    {
+        // First compute the new indicides of the buffer views
+        std::map<int, int> bufferViewIndex;
+        int currentOffset = 0;
+        for (int i = 0, n = m_bufferViews.size(); i < n; i++) {
+            if (indices.count(i) != 0) {
+                currentOffset++;
+            }
+            bufferViewIndex[i] = i - currentOffset;
+        }
+
+        // Fix them in the accessors
+        // TODO sparse accessors
+        for (int i = 0; i < m_accessors.size(); i++) {
+            auto acc = m_accessors[i].toObject();
+
+            auto it = acc.find(GLTF2Import::KEY_BUFFERVIEW);
```

Comment:
Probably nicer to use const QJsonValue v = acc.value(KEY_BUFFERVIEW); if (!v.isUndefined()) ...